react/components/Onboarding/components/Welcome/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from 'react/components/UI/Text';
+import CTAButton from 'react/components/Onboarding/components/UI/CTAButton';
+
+const Welcome = props => {
+  const { goForward } = props;
+
+  return (
+    <div>
+      <Text f={9} my={4}>
+        Welcome to Are.na!
+      </Text>
+      <Text f={9} my={4}>
+        Let's make your first channel.
```

Comment:
Simple thing but use the unicode rsquo: `’` instead of the feet glyph.

Reply:
@danpia @csherron When we prepare copy can we make sure we have the correct chars (I tend to have to remember to replace these things.)

Reply:
Good point, should also be the case for existing copy, about page, etc. ccing leo
For curly single quote: `Alt + Shift + ]`

Reply:
(I can never remember it because the html entities are burned into my memory and always copy paste from this network of sites http://www.rsquo.net/)

Reply:
👍